corehq/apps/style/static/style/js/hq_extensions.jquery.js

```diff
@@ -44,9 +44,43 @@
     $.fn.closest_form = function () {
         return this.closest('form, .form');
     };
+
     $.fn.my_serialize = function () {
         var data = this.find('[name]').serialize();
         return data;
     };
 
+    $.fn.addSpinnerToButton = function () {
+        $(this).prepend('<i class="fa fa-refresh fa-spin icon-refresh icon-spin"></i> ');
+    };
+
+    $.fn.removeSpinnerFromButton = function () {
+        $(this).find('i').remove();
+    };
+
+    $.fn.disableButtonNoSpinner = function () {
+        $(this).attr('disabled', 'disabled')
+            .addClass('disabled');
+    };
+
+    $.fn.disableButton = function () {
+        $(this).disableButtonNoSpinner();
+        $(this).addSpinnerToButton();
+    };
+
+    $.fn.enableButton = function () {
+        $(this).removeSpinnerFromButton();
+        $(this).removeClass('disabled')
+            .removeAttr('disabled');
+    };
+
+    $.fn.koApplyBindings = function (context) {
```

Comment:
Haven't been part of the discussions for this, so I might be overthinking this -- does using require not allow us to be much more selective than having to import whole util files when we want to just use a single function that is within it? I'm thinking of this one (`koApplyBindings`) in particular -- it seems that this will be used in most places we use knockout, where I might not care about disabling buttons with spinners. 
Basically, I'm wondering if it is worth breaking these files out more so we are more explicit in what exactly we are using?